tests/_support/Step/Acceptance/Admin/CampaignImportSteps.php

```diff
@@ -0,0 +1,84 @@
+<?php
+namespace Step\Acceptance\Admin;
+
+use Page\Acceptance\Admin\GutenbergEditor;
+
+/**
+ * Gutenberg: steps to use Gutenberg editor
+ */
+class CampaignImportSteps
+{
+  /**
+   * @var \AcceptanceTester;
+   */
+  protected $tester;
+
+  public function __construct(
+    \AcceptanceTester $I
+  ) {
+    $this->tester = $I;
+  }
+
+  /**
+   * @example I select xml file to import "test-campaign.lorem-ipsum.xml"
+   *
+   * @When I select xml file to import :arg1
+   */
+  public function iSelectXmlFileToImport($arg1)
+  {
+    $I = $this->tester;
+    $I->attachFile( '.wp-upload-form > p > #upload', $arg1 );
+  }
+
+  /**
+   * @When I upload file and import
+   */
+  public function iUploadFileAndImport()
+  {
+    $I = $this->tester;
+    $I->click( '.submit > #submit' );
+  }
+
+  /**
+   * @example I checked checkbox "#import-attachments"
+   *
+   * @When I checked checkbox :arg1
+   */
+  public function iCheckedCheckbox($arg1)
+  {
+    $I = $this->tester;
+    $I->waitForElement( $arg1, 5);
+    $I->checkOption( $arg1 );
+  }
+
+  /**
+   * @When I submit the import form
+   */
+  public function iSubmitTheImportForm()
+  {
+    $I = $this->tester;
+    $I->click( '.submit > .button' );
+  }
+
+  /**
+   * @example I see a import status message "All done. Have fun!".
+   *
+   * @Then I see a import status message :arg1
+   */
+  public function iSeeAImportStatusMessage(?string $message): void
+  {
+    $I = $this->tester;
+    $I->waitForText( 'All done.', 5);
```

Comment:
You should use your `$message`, or you could also change the step to something without parameter, like `I see a successful import message`